src/components/PasswordSetterBox.js

```diff
@@ -56,18 +56,18 @@ class PasswordSetterBox extends Nimiq.Observable {
 
         /* eslint-disable max-len */
         $el.innerHTML = TemplateTags.hasVars(1)`
-            <div class="password-strength strength-short  nq-text-s" data-i18n="passwordbox-password-strength-short" >Enter at least 8 characters</div>
-            <div class="password-strength strength-weak   nq-text-s" data-i18n="passwordbox-password-strength-weak"  >That is a weak password</div>
-            <div class="password-strength strength-good   nq-text-s" data-i18n="passwordbox-password-strength-good"  >Ok, that is an average password</div>
-            <div class="password-strength strength-strong nq-text-s" data-i18n="passwordbox-password-strength-strong">Great, that is a strong password</div>
-            <div class="password-strength strength-secure nq-text-s" data-i18n="passwordbox-password-strength-secure">Super, that is a secure password</div>
+            <div class="password-strength strength-short  nq-text-s" data-i18n="passwordbox-password-strength-short" >Enter min. 8 characters</div>
+            <div class="password-strength strength-weak   nq-text-s" data-i18n="passwordbox-password-strength-weak"  >Sufficient password</div>
+            <div class="password-strength strength-good   nq-text-s" data-i18n="passwordbox-password-strength-good"  >Good password</div>
+            <div class="password-strength strength-strong nq-text-s" data-i18n="passwordbox-password-strength-strong">Strong password</div>
+            <div class="password-strength strength-secure nq-text-s" data-i18n="passwordbox-password-strength-secure">Secure password</div>
             <div class="repeat-long nq-text-s" data-i18n="passwordbox-repeat-password-long">No match, please try again</div>
             <div class="repeat-short nq-text-s" data-i18n="passwordbox-repeat-password-short">Password is too short</div>
             <div class="repeat-password nq-text-s" data-i18n="passwordbox-repeat-password">Repeat your password</div>
 
             <div password-input></div>
 
-            <button class="submit" data-i18n="passwordbox-repeat">Repeat password</button>
+            <button class="submit" data-i18n="passwordbox-confirm">Confirm</button>
```

Comment:
This is the button text during the first password input, yes? I think this should still say "repeat password", as that is the next step. Seeing "confirm", I would not expect another input, but that I _confirm_ the password with this button (but instead I have to re-enter it).

I expect the word "confirm" to _end_ the input.

@julianbauer

Reply:
The difference here is what this button means – does it describe the action that a user performs by clicking it or does it describe where we sent the user after he/she clicked it. Still, you have a very valid point that this might lead to a false expectation that we disappoint a bit, so let's stay with the 'Repeat Password'.